File: BoxTetris/src/main/java/boxtetris/entities/Cuboid.java
package boxtetris.entities;

public class Cuboid extends Dimensions {

    private Integer weight;

    public Cuboid(Integer length, Integer width, Integer height, Integer weight) {
        super(length, width, height);
        this.weight = weight;
    }

    public void rotate() {
        Integer oldLength = getLength();
        setLength(getWidth());
        setWidth(oldLength);
    }

    /**
     * @return Integer
     */
    public Integer getWeight() {
        return weight;
    }
}

File: frontend/src/components/Cuboid.js
import React from 'react';

const Cuboid = (cuboid) => {
  return (
    <tr>
      <td>{cuboid.name}</td>
      <td>{cuboid.length}</td>
      <td>{cuboid.width}</td>
      <td>{cuboid.height}</td>
      <td>{cuboid.weight}</td>
      <td>{cuboid.demand}</td>
    </tr>
  );
};

export default Cuboid;
